api/topology/src/main/java/quarks/topology/TStream.java

```diff
@@ -443,7 +443,7 @@ else if (WARNING.equals(lr.getLevel()))
     Set<String> getTags(); 
     
     /**
-     * Add an alias for the stream.
+     * Set an alias for the stream.
      * <p>
      * The alias must be unique within the topology.
```

Comment:
https://issues.apache.org/jira/browse/QUARKS-154 is for "type" naming conventions doc/improvements.